src/index.html

```diff
@@ -11,6 +11,29 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="conteiner">
+      <div class="inputDiv" >
+        <img src="/images/Search.svg"
+          alt="search"
+          class="img"
+          >
+        <input
+          class="input"
+          data-qa="big"
+          type="text"
+          placeholder="Try &quot;Los Angeles&quot;"
+        >
+      </div>
+      <div class="inputDiv smallDiv">
+        <img src="/images/Search.svg"
+          class="img--small"
+          alt="search">
```

Comment:
```suggestion
        <img src="/images/Search.svg"
          class="img--small"
          alt="search"
        >
```
переноси закрывающую скобку тега на новую строку

Reply:
Тут вообще нужен тег img? Мне кажется можно сделать через background, так как это изображение не несет никакой смысловой нагрузки